index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+   <meta charset="UTF-8">
+   <meta http-equiv="X-UA-Compatible" content="IE=edge">
+   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   <title>Document</title>
+</head>
+
+
+
+<body style="margin-left: 30px">
+   <h1>Історія Львова</h1>
+   <img src="XIX.jpg" alt="">
+   <hr>
+   <div>
+      <p style="font-size: 14px;"><b>Львів</b> - найбільше місто Західної України,що протягом багатовікової історії
+         було,і є нині,науковим,культурним та національним центром регіону.</p>
+      <p><span style="font-style: italic;font-weight: bold;"> В найдавніші часи</span> - столиця Галицько-Волинської
```

Comment:
you can use special tags for bold and italic font - `<i></>` and `<b></>` instead of inline styles